pkg/cmd/grafana-cli/commands/install_command.go

```diff
@@ -97,7 +97,9 @@ func InstallPlugin(pluginName, version string, c utils.CommandLine, client utils
 
 		// Plugins which are downloaded just as sourcecode zipball from github do not have checksum
 		if v.Arch != nil {
-			checksum = v.Arch[osAndArchString()].SHA256
+			if configuredChecksum := v.Arch[osAndArchString()].SHA256; len(configuredChecksum) > 0 {
+				checksum = configuredChecksum
```

Comment:
This is what I had in mind, try specific arch first and eventually fall back to "any":
```suggestion
			archMeta, exists := v.Arch[osAndArchString()]
			if !exists {
				archMeta = v.Arch["any"]
			}
			checksum = archMeta.SHA256
```

Reply:
this approach make sense, if we get to this point then grafana-com must have returned either the requested arch (if a package for that arch exists) or the "any" fallback

Reply:
FWIW, I tested this modification before with both "linux-amd64" and "any" arch plugins, and found that it works.

Reply:
What i have as impression is that we took already sha256 from any, if it is the case, the specific architecture when present could just overwrite the configuration, why we need to specifically fallback? maybe there is something i missed.